src/editor/contentState/selectionCtrl.js

```diff
@@ -0,0 +1,28 @@
+import selection from '../selection'
+
+const selectionCtrl = ContentState => {
+  ContentState.prototype.selectionHandler = function (event) {
+    const { start } = this.cursor
+    const block = this.getBlock(start.key)
+    if (/th|td/.test(block.type)) {
+      event.preventDefault()
+      console.log('table detected!') // TODO: #DEBUG
+
```

Comment:
@fxha You need to set the `this.cursor` firstly.

```
this.cursor = {
    start: { key: start.key, offset: 0 },
    end: { key: start.key, offset: block.text.length}
}
// then after other codes
this.partialRender()
```

code above just a example. it maybe help you? If you have any other questions about `editor-core`, I am very happy to help you！